langkit/gdb/debug_info.py

```diff
@@ -8,7 +8,12 @@
 import inspect
```

Comment:
That's a cosmetic comment, but the naming scheme becomes mildly unclear IMHO. We can maybe see that when we have to rework this for the concrete syntax, but the part that are basically "source maps" go beyond gdb (that's the point of this commit), so having those in a module like `dbg_info.source_maps` rather than `gdb.debug_info` would make more sense IMHO (and then keep the purely gdb part in the `gdb` module)

Reply:
I agree it woud make more sense. I’ll do that in other commits.